src/sqlfluff/dialects/dialect_hive.py

```diff
@@ -154,22 +154,25 @@ class CreateDatabaseStatementSegment(BaseSegment):
 
 @hive_dialect.segment(replace=True)
 class CreateTableStatementSegment(BaseSegment):
-    """A `CREATE TABLE` statement."""
+    """
+    A `CREATE TABLE` statement. Full Apache Hive `CREATE TABLE` reference here:
```

Comment:
I think this needs to be like this, but could be wrong:

```suggestion
    """A `CREATE TABLE` statement.
    
    Full Apache Hive `CREATE TABLE` reference here:
```

Let's see if the CI fails and, if so, change it to that.

Reply:
Yup, as I thought linting failed:

```
./src/sqlfluff/dialects/dialect_hive.py:157:1: D205 1 blank line required between summary line and description
./src/sqlfluff/dialects/dialect_hive.py:157:1: D212 Multi-line docstring summary should start at the first line
./src/sqlfluff/dialects/dialect_hive.py:157:1: D415 First line should end with a period, question mark, or exclamation point
```

Will commit the above change and then merge.